ansible/roles/proteus-events/tasks/main.yml

```diff
@@ -0,0 +1,63 @@
+- name: create Proteus group
+  group: name={{ proteus_group }}  state=present
+
+- name: create Proteus user
+  user:
+    name: "{{ proteus_user }}"
+    group: "{{ proteus_group }}"
+    createhome: no
+    shell: /sbin/nologin
+    comment: "Proteus User"
+    state: present
+
+- name: set permissions, owner and group
+  file:
+    path: "{{ proteus_install_path }}"
+    state: directory
+    owner: "{{ proteus_user }}"
+    group: "{{ proteus_group }}"
+    mode: "go-w"
+    recurse: yes
+
+- name: mkdir for config and data
+  file:
+    path: "{{ item }}"
+    state: directory
+    owner: "{{ proteus_user }}"
+    group: "{{ proteus_group }}"
+    mode: "u=rwx,g=rx,o="
+  with_items:
+    - "{{ proteus_data_path }}"
+    - "{{ proteus_config_path }}"
+
+- name: untar proteus tarball
+  unarchive:
+    src: "{{ proteus_tarball_url }}"
+    dest: "{{ proteus_install_path }}"
+    mode: 0755
+    owner: "{{ proteus_user }}"
+    group: "{{ proteus_group }}"
+    copy: no
```

Comment:
@bassosimone  please, add sha256sums file to releases of proteus binaries and the script that generates releases.

These checksum files may be used to speedup re-run of the playbook as `ger_url` module accepts `checksum` argument [like that](https://github.com/TheTorProject/ooni-sysadmin/blob/ca26b09c46d3c36006f77cbaa866f00654977970/ansible/roles/node_exporter/tasks/main.yml#L19).